tools/check_format.py

```diff
@@ -145,8 +145,17 @@ def checkBuildLine(line, file_path, reportError):
   if not whitelistedForProtobufDeps(file_path) and '"protobuf"' in line:
     reportError("unexpected direct external dependency on protobuf, use "
                 "//source/common/protobuf instead.")
+  if '@envoy//' in line:
```

Comment:
Hmm. Actually, I just realized this is going to cause issues for people that are using the formatter in a parent repo along with envoy itself. In a parent repo, this is valid. Can we potentially add a CLI formatter option to skip this check? WDYT?

Reply:
How about I just scope this particular check to filenames underneath envoy/ ?  Would that be sufficient?

Reply:
But then how would it work for the OSS CI?

Reply:
I see...in the docker context the envoy directory piece is chrooted out of existence.  OK, adding a switch.